accord-core/src/main/java/accord/local/KeyHistory.java

```diff
@@ -16,17 +16,20 @@
  * limitations under the License.
  */
 
-package accord.burn.random;
+package accord.local;
 
-import accord.utils.RandomSource;
-
-public interface RandomInt extends RandomLong
+/**
+ * For operations that need information on historical operations, this indicates the
+ * amount of data needed.
+ */
+public enum KeyHistory
 {
-    int getInt(RandomSource randomSource);
+    DEPS,
+    ALL,
```

Comment:
I'm not sure these names are very intuitive. Perhaps we should just call them e.g. FOR_COMPUTING_DEPS and FOR_RECOVERY?

Reply:
I think this may have gotten lost?